modules/core/src/main/java/org/apache/synapse/aspects/flow/statistics/collectors/CloseEventCollector.java

```diff
@@ -75,6 +74,12 @@ public static void closeEntryEvent(MessageContext messageContext, String compone
 			}else {
 				addEventAndDecrementCount(messageContext, closeEvent);
 			}
+
+			if (isOpenTracingEnabled()) {
+				OpenTracingManagerHolder.getOpenTracingManager().getHandler().
+						handleCloseEntryEvent(statisticDataUnit, messageContext);
+			}
```

Comment:
Let's create a util function for this. it's reused more than 16 times

Reply:
Each of those usages is a separate type of event (eg: [`handleCloseEntryEvent()`](https://github.com/wso2/wso2-synapse/pull/1466/files#diff-737675a6275c53cad6d04385403f3f62R80), [`handleCloseForcefully()`](https://github.com/wso2/wso2-synapse/pull/1466/files#diff-737675a6275c53cad6d04385403f3f62R109), [`handleOpenEntryEvent()`](https://github.com/wso2/wso2-synapse/pull/1466/files#diff-12af4a45125f25d590dc29932fb52f0cR113) and so on), so we can't re-use a util function.